client/src/components/card/style.css

```diff
@@ -0,0 +1,65 @@
+.card-container {
+  max-width: 300px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #b3b3b3;
```

Comment:
podobno uwaga z tymi kolorami. Jak załatwisz kolory na poprzednim branchu to zmerguj sobie go tutaj i skorzystaj wtedy z globalnych kolorów